nss-day-cohort-52/rare-client-smoky-pythons
# Lz search by title

### Issue ticket(s): #46

### New file(s): None

### Modified File(s): PostList.js, PostRoutes.js, ApplicationViews.js, PostsRepository.js

### Modifications: PostList.js - created input tag and function to capture value. 
PostRoutes.js - added props to PostList function call. 
ApplicationViews.js - created titleFilter useState function, if statement to search by title, added props to PostRoutes function call. 
PostsRepository.js - added a searchByTitle fetch call to capture value input. 

### Steps to test:
1. While in the terminal, use git fetch --all in the root directory of this repo
2. Checkout out to lz-searchByTitle
3. npm start client 
4. check to see if post list rerenders with correct posts after typing a string into the title input field.
